src/main/resources/db/changelog/changes/3-create-info-user-table-changelog.sql

```diff
@@ -0,0 +1,23 @@
+
+--liquibase formatted sql
+
+--changeset noyan:1
+
+CREATE TABLE `users` (
+                         `id` bigint NOT NULL AUTO_INCREMENT,
+                         `create_date` datetime(6) DEFAULT NULL,
```

Comment:
please change datetime types to timestamp.

Reply:
I just changed datetime types to timestamp.